deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
+SCRIPTPATH=$(pwd -P)
+FILE1="docker-compose-base.yaml"
+FILE2="docker-compose-disk.yaml"
+FILE3="disk.env"
+
```

Comment:
Once thing I usually do at the start of any shell script that uses programs not found in `/bin` or `/sbin` is I check that they're installed. Something like this:

```
exit_error() {
   printf "%s" "$1" >&2
   exit 1
}

for cmd in ncat grep tr dd sed cut date sqlite3 awk xxd; do
   command -v "$cmd" >/dev/null 2>&1 || exit_error "Missing command: $cmd"
done
```

Reply:
Good advice. Added.